### PR TITLE
Accordions

### PR DESCRIPTION
**Pull request by djsaun**
New pull request for the FAQ component. Accidentally had it merging into the master branch before.

Add ```<link rel="stylesheet" href="../../../dist/faqs/faqs.css">``` to the src/components/faqs.html file to view the component locally.

### CODE REVIEW

**Comment by fleeting** [comment]
@defvayne23 @djsaun Should this just be called something like `basic accordions` or something better? In mbCMS for the PageBuilder we don't call it FAQ but instead just Accordions since ultimately it doesn't need to be questions and answers but anything collapsable.

**Comment by defvayne23** [comment]
I was very confused at what FAQs were at first. Was wondering what made it different from other accordions.

**Comment by djsaun** [comment]
I'll update everything from FAQ to accordion.

**Comment by djsaun** [comment]
FAQs are now Accordions.

Please add ```<link rel="stylesheet" href="../../../dist/accordions/accordions.css">``` to the src/components/accordions.html file to view the component locally.

File: src/components/accordions/usage.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+* Import the CSS into your project -> `@import "/bower_components/jubilee/dist/accordions/accordions.css";`.
+* Reference `/bower_components/jubilee/src/components/accordions/accordions.html` for HTML structure.
+
+## Customize
+
+Default styles are included but you should override with styles that match the project. Comments have been added to the source css file indicating how to target individual elements.
+
+```
+/* Setting active border color. */
+.faqs > .faq-item {
```
Comment (fleeting) [inline src/components/accordions/usage.md]: Need to update this to the new classes.

Comment (djsaun) [reply src/components/accordions/usage.md]: Good catch. This has been fixed.

**Review by fleeting — APPROVED** [review]

**Review by fleeting — CHANGES_REQUESTED** [review]
We should moved the padding to be on the `label` instead of the `.accordion-item` to make the entire area clickable.


Tested on Chrome, iOS Chrome, Firefox, IE11, and Edge 14 - all good.

**Comment by djsaun** [comment]
Moved the padding from ```.accordion-item``` to ```label```.

**Comment by fleeting** [comment]
@djsaun Need to tweak the `+` to show back up. It should be removing `width: 100%;` from `.accordion-item label` and repositioning.

**Comment by djsaun** [comment]
Icon issue should be fixed.

**Review by fleeting — APPROVED** [review]

**Pull request merged**